_layouts/default.html

```diff
@@ -29,13 +29,13 @@
     <meta name="twitter:description" content="{{ social_description | strip_newlines | truncate: 150 | strip_html }}">
     <meta name="twitter:creator" content="@OmbuLabs">
     <!-- Twitter summary card with large image must be at least 280x150px -->
-    <!-- meta name="twitter:image:src" content="http://www.example.com/image.html" -->
+    <meta name="twitter:image:src" content="{{site.domain_name}}/assets/profile.png">
 
     <!-- Open Graph data -->
     <meta property="og:title" content="{{ page_title }}" />
     <meta property="og:type" content="article" />
     <meta property="og:url" content="{{site.domain_name}}{{site.baseurl}}{{ page.url }}" />
-    <meta property="og:image" content="{{site.domain_name}}/assets/isotype.png" />
+    <meta property="og:image" content="{{site.domain_name}}/assets/profile.png" />
```

Comment:
@ercohen14 I think this is trying to use the `ombulabs.com` domain only, it should be:

`{{site.baseurl }}/assets/images/profile.png`

in order to reach https://www.ombulabs.com/blog/assets/images/profile.png

And same for the meta tag above.